packages/salesforcedx-vscode-apex/test/jest/views/settings/configurations.test.ts

```diff
@@ -0,0 +1,33 @@
+import * as sinon from 'sinon';
```

Comment:
So your jest unit test files should 1-1 map to the code under test.  I would expect to see a `jest/languageServer.test.ts` file in this PR with tests the cover the code additions there.